client/azure/azure.go

```diff
@@ -0,0 +1,132 @@
+package azure
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+
+	"github.com/Azure/azure-sdk-for-go/arm/resources/resources"
+	"github.com/Azure/go-autorest/autorest"
+	"github.com/Azure/go-autorest/autorest/adal"
+	"github.com/Azure/go-autorest/autorest/azure"
+)
+
+// Config contains the common attributes to create an Azure client.
+type Config struct {
+	// Dependencies
```

Comment:
smaller nitpick: fullstop missing
bigger nitpick: there should be a empty line between the comment and a field definition. Otherwise it's the field comment in godoc.